### PR TITLE
doc: nrf91 series doc restructure

### PR DESCRIPTION
**Pull request by martiles**
Restructuring "Working with nrf91 series" to
have a similar structure as "Working with nrf52/53".

Signed-off-by: Martin Lesund <martin.lesund@nordicsemi.no>

### CODE REVIEW

**Comment by martiles** [comment]
Fyi. This commit is a proposal on how we could restructure the doc. page.
(would make sense to remove the "ug_thingy91.rst" since that is covered in this now)

File: doc/nrf/ug_nrf91.rst [hunk 1]
```diff
@@ -3,11 +3,17 @@
 Working with nRF91 Series
 #########################
 
-The |NCS| provides support for developing on the nRF9160 System in Package (SiP) using the nRF9160 DK and the Nordic Thingy:91.
+.. contents::
+   :local:
+   :depth: 2
+
+The |NCS| provides support for developing on the `nRF9160 System in Package (SiP) <nRF9160 product website_>`_ using the `nRF9160 DK (PCA10090) <nRF9160 DK product page_>`_ and `Thingy:91 (PCA20035) <Thingy\:91 product page_>`_, and offers :ref:`samples <nrf9160_samples>` dedicated to these devices.
```
Comment (coderbyheart) [inline doc/nrf/ug_nrf91.rst:10]:
```suggestion
The |NCS| provides support for developing on the `nRF9160 System in Package (SiP) <nRF9160 product website_>`_ using the `nRF9160 DK (PCA10090) <nRF9160 DK product page_>`_ and the `Thingy:91 (PCA20035) <Thingy\:91 product page_>`_, and offers :ref:`samples <nrf9160_samples>` dedicated to these devices.
```

File: doc/nrf/ug_nrf9160.rst [hunk 2]
```diff
@@ -7,34 +7,34 @@ Working with nRF9160 DK
...
+****************
 
-For more details on the SiP, see the `nRF9160 product website`_ and the `nRF9160 Product Specification`_.
+The nRF9160 DK contains an nRF52840 SoC that is used to route some of the nRF9160 SiP pins to different components on the DK, such as LEDs and buttons, and to specific pins of the nRF52840 SoC itself.
+For a complete list of all the routing options available, see the `nRF9160 DK board control section in the nRF9160 DK User Guide`_.
 
-.. figure:: images/nrf9160_ug_overview.svg
-   :alt: Overview of nRF91 application architecture
+The nRF52840 SoC on the DK comes preprogrammed with a firmware.
+If you need to restore the original firmware at some point, download the nRF9160 DK board controller FW from the `nRF9160 DK product page`_.
```
Comment (coderbyheart) [inline doc/nrf/ug_nrf9160.rst:23]:
```suggestion
If you need to restore the original firmware at some point, download the nRF9160 DK board controller firmware from the `nRF9160 DK product page`_.
```

for consistency

File: doc/nrf/ug_nrf9160.rst [hunk 3]
```diff
@@ -7,34 +7,34 @@ Working with nRF9160 DK
...
+To change the routing options, enable or disable the corresponding devicetree nodes for that board as needed.
+See :ref:`zephyr:nrf9160dk_board_controller_firmware` for detailed information.
 
-The figure illustrates the conceptual layout when targeting an nRF9160 Cortex-M33 application MCU with TrustZone.
+Modem firmware
+**************
 
-Application MCU
-===============
+You can update the modem firmware in an nRF9160 DK by multiple methods. See :ref:`Updating the nRF9160 cellular modem <nrf9160_update_modem_fw>` for more information.
```
Comment (coderbyheart) [inline doc/nrf/ug_nrf9160.rst:34]:
```suggestion
You can update the modem firmware in an nRF9160 DK with multiple methods. See :ref:`Updating the nRF9160 cellular modem <nrf9160_update_modem_fw>` for more information.
```

**Review by coderbyheart — APPROVED** [review]

**Comment by martiles** [comment]
Thank you @coderbyheart!

**Please hold off any more reviews at the moment.** 
@umapraseeda will make a new PR which includes a lot of fixes and we'll close this one down.

File: doc/nrf/ug_nrf91_features.rst [hunk 4]
```diff
@@ -0,0 +1,309 @@
...
+     You can hardcode the information in the application, or you can use functionality like AWS jobs to provide the URL dynamically.
+
+The full FOTA procedure depends on where the binary files are hosted for download.
+
+You can refer to the following implementation samples and applications:
+
+* :ref:`http_full_modem_update_sample` sample - performs a full firmware OTA update of the modem.
+* :ref:`http_modem_delta_update_sample` sample - performs a delta OTA update of the modem firmware.
+* :ref:`http_application_update_sample` sample - performs a basic application FOTA update.
+* :ref:`aws_fota_sample` sample - performs a FOTA update via MQTT and HTTP, where the firmware download is triggered through an AWS IoT job.
```
Comment (martiles) [inline doc/nrf/ug_nrf91_features.rst:307]: Include:
* :ref:`azure_fota_sample` sample - performs a FOTA update from the Azure IoT Hub

**Review by jhn-nordic — APPROVED** [review]

**Review by Rallare — APPROVED** [review]

**Review by simensrostad — APPROVED** [review]

**Review by umapraseeda — APPROVED** [review]

**Pull request merged**